data.sql

```diff
@@ -6,3 +6,13 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts,
 ('Boarmon', '2005-06-07', 20.4, TRUE, 7, NULL),
 ('Blossom', '1998-10-13', 17, TRUE, 3, NULL),
 ('Ditto', '2022-05-14', 22, TRUE, 4, NULL);
+
+INSERT INTO owners (full_name, age) VALUES 
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
```

Comment:
- [ ] Dear @bohaz, according to the requirements inside the `data.sql` there are some queries you have missed. 
> 1. if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead
> 2. all animals got [assigned owners](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Modify%20your%20inserted%20animals%20to,Winchester%20owns%20Angemon%20and%20Boarmon.)

**Please add this informations**

Reply:
Thanks, @TadesseDev for your review!

- I have updated my schema.sql and data.sql files.